### DISCUSSION TITLE
Express Server with static assets like index.html

### DISCUSSION THREAD

**Discussion by Bennch** [yao-pkg/pkg#46] [General]
Hello,

I have a very disappointing problem. I guess the answer is easy, but i don't find the way out.

I create my app as follow:

`pkg --compress Brotli -t node20-macos-x64 app.js --assets src/gui/**/*`

I think that my assets on the src/gui folder is now build in the exutable. 

How can i serve this bundled files now?

```
app.use(express.static(__dirname));
app.use('/css', express.static('src/gui/css'));
app.use('/img', express.static('src/gui/img'));
app.use('/js', express.static('src/gui/js'));

app.get('/', isAuthenticated, (req, res) => {
    res.sendFile('index.html', { root: path.join(__dirname, './src/gui') });
});
```

The above did not work.. 

To be clear. If i add the src/gui folder in the same location as the app.exe it works, but i tought that static files are directely into the app.exe. So how can i serve them? 

Thank you!

**Comment by robertsLando** [comment]
@Bennch `--assets` doesn't exist, you should declare assets in package.json `pkg` section

**Reply by Bennch** [reply]
Ok, thank you. Now I'm a step further. 

```
  "pkg": {
    "scripts": [ 
    "src/*", 
    "app.js"
   ],
    "assets": [ 
    "src/gui/css/*", 
    "src/gui/img/*",
    "src/gui/js/*",
    "src/gui/index.html",
    "src/gui/login.html" ],
    "outputPath": "./"
  },
```

But how i serve this packed files now correctly? The above exaple dont work.

I get 

`Error: ENOENT: no such file or directory, stat 'C:\Users\**\Bot\build\src\gui\login.html'`

It looks for the packaged files in its own directory instead of /snapshot/

Thank you!

**Reply by robertsLando** [reply]
Use `__dirname` and it will resolve to snapshot
